test/functional/rpc_psbt.py

```diff
@@ -189,10 +189,46 @@ def run_test(self):
         assert_equal(walletprocesspsbt_out['complete'], True)
         self.nodes[1].sendrawtransaction(self.nodes[1].finalizepsbt(walletprocesspsbt_out['psbt'])['hex'])
 
-        self.log.info("Test feeRate of 0.1 BTC / KB produces a total fee slightly below -maxtxfee (~0.05280000)")
+        self.log.info("Test feeRate of 0.1 BTC / KB produces a total fee at or slightly below -maxtxfee (~0.05290000)")
         res = self.nodes[1].walletcreatefundedpsbt(inputs, addr, 0, {"feeRate": 0.1, "add_inputs": True})
         assert_approx(res["fee"], 0.055, 0.005)
 
+        self.log.info("Test passing walletcreatefundedpsbt explicit feerate with conf_target and estimate_mode")
+        for unit, feerate in {"btc/kb": 0.1, "sat/b": 10000}.items():
```

Comment:
Probably not the right place to bring this up, but the explicit specifying of the unit does seem really error prone as mentioned in the comments of #19543 before.

Reply:
Yes. It would be good to replace this with the fixed-unit `feerate_sat_vb` argument before the 0.21 branch-off.